Popup/Popup.module.less

```
@@ -92,29 +97,63 @@
 			max-width: 100%;
 		}
 
+		&.fullscreen,
+		&.left,
+		&.right {
+			justify-content: @agate-popup-justify-content;
+
+			.body {
+				height: @agate-popup-body-height;
+			}
+		}
+
 		&.bottom {
 			border-radius: @agate-popup-bottom-border-radius;
+
+			.buttons {
+				border-radius: @agate-popup-bottom-buttons-border-radius;
+			}
 		}
 
 		&.left {
 			border-radius: @agate-popup-left-border-radius;
+
+			.buttons {
+				border-radius: @agate-popup-left-buttons-border-radius;
+			}
 		}
 
 		&.fullscreen {
 			border-radius: @agate-popup-fullscreen-border-radius;
+
+			.buttons {
+				border-radius: @agate-popup-fullscreen-buttons-border-radius;
+			}
 		}
 
 		&.right {
 			border-radius: @agate-popup-right-border-radius;
+
+			.buttons {
+				border-radius: @agate-popup-right-buttons-border-radius;
+			}
 		}
 
 		&.top {
 			border-radius: @agate-popup-top-border-radius;
 		}
 
 		.title {
+			width: @agate-popup-title-width;
+			height: @agate-popup-title-height;
 			color: @agate-popup-text-color;
+			font-size: @agate-popup-title-font-size;
 			padding: @agate-popup-title-padding;

```

We have proper mixin for what you intended.
```suggestion
			.padding-start-end(@agate-popup-title-padding);

```